### ISSUE TITLE
Visualization bug: Error if no shooting moves?

### ISSUE THREAD

**Issue by dwhswenson** [choderalab/openpathsampling#338]
Separate issue, following on #337: visualization raises errors if there are no shooting moves in the sample list.

Simple example:

```python
import openpathsampling as paths
from openpathsampling.visualize import PathTreeBuilder

storage = paths.storage.Storage("mstis.nc")

# take any sample; here the initial sample for replica 0
samp = storage.steps[0].active[0]

tree = PathTreeBuilder(storage)
tree.rejected = True
tree.show_redundant = False
tree.states = []

tree.from_samples([samp])
```

Output:
```
---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-5-b6d7f6dae0d7> in <module>()
----> 1 tree.from_samples([samp])

/Users/dwhs/GoogleDrive/msm-tis/openpathsampling/visualize.pyc in from_samples(self, samples, clear)
   1161                 self.renderer.label(self.renderer.shift_x + 5.0, 1 + tc, 1, str(
   1162                     self.storage.idx(s)) , align='end',
-> 1163                                     color=font_color)
   1164             )
   1165 

UnboundLocalError: local variable 'font_color' referenced before assignment
```

From playing around with which samples are included, it looks like this error occurs when there are no shooting moves in that list of samples. Once there is at least one shooting move, it behaves as expected.

**Comment by jhprinz** [comment]
Will make a fix...

Anyway we should overhaul the pathmovevis as well. More and cleaner options, etc...

**Comment by dwhswenson** [comment]
Fixed by #341.
